__tests__/api.test.js

```diff
@@ -30,3 +30,26 @@ describe("GET/api/topics", () => {
       });
   });
 });
+
+describe("GET/api", () => {
+  it("status: 200, responds with all endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endPoints).toHaveProperty("GET /api");
+      });
+  });
+  it("should return json format", () => {
+    return request(app).get("/api").expect("Content-Type", /json/);
+  });
+  it("status: 404, returns an error if endpoint doesn't exist", () => {
```

Comment:
This test should be here, but it isn't related to this endpoint. This is a test for a non existing endpoint which should have it's own describe block.